plugins/inputs/snmp/snmp.go

```diff
@@ -9,6 +9,7 @@ import (
 	"math"
 	"net"
 	"os/exec"
+	"regexp"
```

Comment:
@Hipska maybe in this cleanup you can include fixing linters findings?

```
plugins/inputs/snmp/snmp.go:1:9                  revive       import-shadowing: The name 'snmp' shadows an import name
plugins/inputs/snmp/snmp.go:8:2                  revive       imports-blacklist: should not use the following blacklisted import: "log"
plugins/inputs/snmp/snmp.go:376:27               revive       flag-parameter: parameter 'walk' seems to be a control flag, avoid control coupling
plugins/inputs/snmp/snmp.go:406:21               revive       flag-parameter: parameter 'walk' seems to be a control flag, avoid control coupling
plugins/inputs/snmp/snmp.go:594:15               unconvert    unnecessary conversion
plugins/inputs/snmp/snmp.go:692:10               revive       early-return: if c {...} else {... return } can be simplified to if !c { ... return } ...
plugins/inputs/snmp/snmp.go:748:1                revive       function-result-limit: maximum number of return results per function exceeded; max 3 but got 5
plugins/inputs/snmp/snmp.go:765:1                revive       function-result-limit: maximum number of return results per function exceeded; max 3 but got 5
plugins/inputs/snmp/snmp.go:836:1                revive       function-result-limit: maximum number of return results per function exceeded; max 3 but got 5
plugins/inputs/snmp/snmp.go:862:1                revive       function-result-limit: maximum number of return results per function exceeded; max 3 but got 5
plugins/inputs/snmp/snmp_mocks_test.go:47:3      revive       unhandled-error: Unhandled error in call to function fmt.Fprintf
plugins/inputs/snmp/snmp_mocks_test.go:48:3      revive       deep-exit: calls to os.Exit only in main() or init() functions
plugins/inputs/snmp/snmp_mocks_test.go:50:2      revive       unhandled-error: Unhandled error in call to function fmt.Printf
plugins/inputs/snmp/snmp_mocks_test.go:51:2      revive       unhandled-error: Unhandled error in call to function fmt.Fprintf
plugins/inputs/snmp/snmp_mocks_test.go:53:3      revive       deep-exit: calls to os.Exit only in main() or init() functions
plugins/inputs/snmp/snmp_mocks_test.go:55:2      revive       deep-exit: calls to os.Exit only in main() or init() functions
plugins/inputs/snmp/snmp_test.go:2:9             revive       import-shadowing: The name 'snmp' shadows an import name
plugins/inputs/snmp/snmp_test.go:15:2            revive       duplicated-imports: Package "github.com/influxdata/telegraf/internal/snmp" already imported
plugins/inputs/snmp/snmp_test.go:476:2           staticcheck  SA5001: should check returned error before deferring srvr.Close()
plugins/inputs/snmp/snmp_test.go:495:16          errcheck     Error return value of `srvr.WriteTo` is not checked
plugins/inputs/snmp/snmp_test.go:515:2           revive       unhandled-error: Unhandled error in call to function srvr.Close
plugins/inputs/snmp/snmp_test.go:526:2           staticcheck  SA5001: should check returned error before deferring srvr.Close()
plugins/inputs/snmp/snmp_test.go:545:16          errcheck     Error return value of `srvr.WriteTo` is not checked
plugins/inputs/snmp/snmp_test.go:565:2           revive       unhandled-error: Unhandled error in call to function srvr.Close
plugins/inputs/snmp/snmp_test.go:748:10          errcheck     Error return value of `s.Gather` is not checked
plugins/inputs/snmp/snmp_test.go:795:10          errcheck     Error return value of `s.Gather` is not checked
```

Reply:
Ouch, I think I'm afraid I'm not able to do that. I could give it a try if really wanted.